other/upgrade_2-0_mysql.sql

```diff
@@ -2778,13 +2778,13 @@ if (file_exists($GLOBALS['boarddir'] . '/Themes/babylon'))
 		smf_mysql_free_result($theme_request);
 		unset($row);
 
-		$known_themes = explode(', ', $modSettings['knownThemes']);
+		$known_themes = explode(',', $modSettings['knownThemes']);
 
 		// Remove this value...
 		$known_themes = array_diff($known_themes, array($id_theme));
 
 		// Change back to a string...
-		$known_themes = implode(', ', $known_themes);
+		$known_themes = implode(',', $known_themes);
```

Comment:
I hope you don't forget postgres,
when you work on files which are 2.x .sql

Reply:
The pg .sql files dont have any of this, for example, deletion of old 1.x themes when checking 2.0.  

I considered trying to build them from scratch, but frankly, I don't know how.  So I can't...

The good news is that the only impacted installs are 1.x pg installs - which I doubt there are any at all.

Reply:
And what is with this here?
https://github.com/SimpleMachines/SMF2.1/blob/e0848c82fbbc912b51789a2fefded6b6830f086f/other/upgrade_2-0_postgresql.sql#L1026-L1028

Reply:
Compare upgrade_2-0_postgresql.sql to upgrade_2-0_mysql.sql....

It's ~1300 lines vs ~3000.   The pg one is woefully incomplete & completely not acceptable for migration from 1.x.  It simply isn't designed to do so today.  As an example, look at the very first set of modifications covered in the mysql version - all the column renames, etc.  NONE of that is covered in the pg version, meaning, _a pg 1.x will not be converted properly to 2.x_.

It appears to me that the pg version - in broad strokes - omits the true "1.x conversion" items but keeps the "2.x validation" items found in the mysql version.  E.g., it will add a table that doesn't exist.  

Before diving in wholesale into pg 1.x to 2.x migration changes, I think we would need to determine 2 things:
 - Are we really trying to support conversion of a pg 1.x forum to a pg 2.x forum?  Does one even exist?
 - What types of changes should be included in this script?